lib/v2/screens/explore_screens/vote_screens/proposals/mappers/proposals_by_scroll_state_mapper.dart

```diff
@@ -1,16 +1,26 @@
 import 'package:seeds/v2/datasource/remote/model/proposals_model.dart';
+import 'package:seeds/v2/datasource/remote/model/support_level_model.dart';
 import 'package:seeds/v2/domain-shared/page_state.dart';
 import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
 import 'package:seeds/v2/i18n/explore_screens/invite/invite.i18n.dart';
 import 'package:seeds/v2/screens/explore_screens/vote_screens/proposals/viewmodels/proposals_list_state.dart';
 import 'package:seeds/v2/screens/explore_screens/vote_screens/vote/interactor/viewmodels/proposal_type_model.dart';
 
+const _alliance = 'alliance';
+const _cmp_funding = 'cmp.funding';
+const _milestone = 'milestone';
+
 class ProposalsByScrollStateMapper extends StateMapper {
```

Comment:
@gguijarro-c-chwy `ProposalsByScrollStateMapper` and `ProposalsStateMapper` They are practically the same except for one instruction. 
Should I remove one and add an `isScroll` parameter to the other?

Reply:
yes

Reply:
Done